pyart/retrieve/qpe.py

```diff
@@ -1,18 +1,644 @@
-#! /usr/bin/env python
+"""
+pyart.retrieve.qpe
+=========================================
+
+Functions for rainfall rate estimation
+
+.. autosummary::
+    :toctree: generated/
+
+    get_lowest_elevation
+    rr_zpoly
+    rr_z
+    rr_kdp
+    rr_a
+    rr_zkdp
+    rr_za
+    rr_hydro
+
+"""
+
 
 import os
 import sys
 from time import time
+from warnings import warn
 
 import numpy as np
 import netCDF4
 import matplotlib as mpl
 import pylab as pl
 
-import pyart.util.met as met
-#import ballsy_masked as ballsy
+from ..util import met
+from ..config import get_metadata, get_field_name, get_fillvalue
+# import ballsy_masked as ballsy
 
 
 def get_lowest_elevation(mmcg_ncf, req_moments, corrections):
     pass
-    #for each of the moments in req_moments fetch the lowest valid return
+    # for each of the moments in req_moments fetch the lowest valid return
+
+
+def rr_zpoly(radar, refl_field=None, rr_field=None):
```

Comment:
More descriptive function names, maybe est_rain_rate_zpoly?  Same with the other function in the module.